src/intTest/java/com/box/sdk/BoxFileIT.java

```diff
@@ -896,6 +896,7 @@ public void uploadLargeFileVersionWithAttributes() throws Exception {
 
     @Test
     public void setsVanityNameOnASharedLink() {
+        long vanityNameRandomSuffix = System.currentTimeMillis();
```

Comment:
Isn't better to create here variable:
`String customVanityName = "myCustomName-" + System.currentTimeMillis()`
and to not repeat it everywhere?